repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/acceptance-tests.yaml

```diff
@@ -1,5 +1,332 @@
 ---
 /content/documents/corporate-website/publication-system/acceptance-tests:
+  /dummy-2:
```

Comment:
move to separate file, plus might want to name it more specifically for what is being tested rather than just Dummy